File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/MovieDetailsActivityModule.java
package com.github.brianspace.moviebrowser.di.modules;

import com.github.brianspace.moviebrowser.ui.activity.MovieDetailsActivity;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

/**
 * Dagger module for MovieDetailsActivity.
 */
@Module
public abstract class MovieDetailsActivityModule {
    @ContributesAndroidInjector
    /* default */ abstract MovieDetailsActivity contributeMovieDetailsActivityInjector();
}

File: app/src/main/java/com/github/brianspace/moviebrowser/di/modules/ui/MovieDetailsActivityModule.kt
package com.github.brianspace.moviebrowser.di.modules.ui

import com.github.brianspace.moviebrowser.ui.activity.MovieDetailsActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

/**
 * Dagger module for MovieDetailsActivity.
 */
@Module
abstract class MovieDetailsActivityModule {
    @ContributesAndroidInjector
    internal abstract fun contributeMovieDetailsActivityInjector(): MovieDetailsActivity
}
